adagrams/game.py

```diff
@@ -1,11 +1,118 @@
+import random
+import string
```

Comment:
This is an unused import. In your IDE, you can tell because it will look grey in color and if you hover over it you should see a message:

![image](https://github.com/user-attachments/assets/50e962b8-84e7-47a3-a259-0afd23bc8147)

Unused imports should be removed before code review.